microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -20,15 +20,30 @@ These approaches are described below.
 
 
 <!-- ====================================================================== -->
-## Navigate to a file URL
+## Selecting an approach
+
+The approaches for loading local content into a WebView2 control support the following scenarios:
+
```

Comment:
![image](https://user-images.githubusercontent.com/45407486/234641799-280db193-06c2-43ef-a59c-42c16e9d8a83.png)

Mac fyi

Reply:
I think its OK? Again open to alternative emojis or 'supported'/'not supported' text if you like.

Reply:
thread can be Resolved